orm/models_fields.go

```diff
@@ -46,9 +47,9 @@ const (
 
 // Define some logic enum
 const (
-	IsIntegerField         = ^-TypePositiveBigIntegerField >> 4 << 5
-	IsPositiveIntegerField = ^-TypePositiveBigIntegerField >> 8 << 9
-	IsRelField             = ^-RelReverseMany >> 14 << 15
+	IsIntegerField         = ^-TypePositiveBigIntegerField >> 5 << 6
+	IsPositiveIntegerField = ^-TypePositiveBigIntegerField >> 9 << 10
+	IsRelField             = ^-RelReverseMany >> 15 << 16
```

Comment:
It looks he did it having in mind the addition of the `TypeTimeField` in the group definition above, so all the indexes will be altered by +1.